src/main/java/core/basesyntax/service/impl/StorageServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.StorageService;
+import core.basesyntax.strategy.CountStrategy;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+import java.util.Map;
+
+public class StorageServiceImpl implements StorageService {
+    private final OperationStrategy operationStrategy;
+    private final Storage storage = new Storage();
```

Comment:
initialize this field in the constructor

Reply:
fixed